src/framework/midi/internal/platform/lin/alsamidiinport.cpp

```diff
@@ -77,7 +77,7 @@ MidiDeviceList AlsaMidiInPort::availableDevices() const
 
     int streams = SND_SEQ_OPEN_INPUT;
     const unsigned int cap = SND_SEQ_PORT_CAP_SUBS_READ | SND_SEQ_PORT_CAP_READ;
-    const unsigned int type_hw = SND_SEQ_PORT_TYPE_PORT | SND_SEQ_PORT_TYPE_HARDWARE;
+    const unsigned int type_hw = SND_SEQ_PORT_TYPE_HARDWARE;
     const unsigned int type_sw = SND_SEQ_PORT_TYPE_PORT | SND_SEQ_PORT_TYPE_SOFTWARE;
```

Comment:
Ok, let's remove it for software too and check that MIDI Thgrouh devices are not on the list (I added type filters in this commit https://github.com/musescore/MuseScore/commit/edd6bb10033864c107d84ea085495c995a6e8bc4)

Reply:
I don't have these MDI Thgrouh devices, even without any filter. Can you check yourself ?
After that, should I update this PR, or you will do the flag removal for software too in another commit ?

Reply:
@eism, can you check if MIDI Thgrouh devices are still not in the list after this patch?

Reply:
Done that in my resurrection of this PR, #27760